components/CloudinaryImage.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image, Transformation } from 'cloudinary-react';
+import path from 'path';
+
+class CloudinaryImage extends React.Component {
+  render() {
+    const { url, alt, maxWidth, aspectRatio } = this.props;
+
+    return (
+      <Image publicId={path.basename(url)} cloud_name="texas-justice-initiative" secure="true" alt={alt}>
```

Comment:
Node has [this neat method](https://nodejs.org/api/path.html#path_path_basename_path_ext) for extracting the file name from a URL!